### ISSUE TITLE
Servercrash in Mesa Biome

### ISSUE THREAD

**Issue by d00mfish** [PaperMC/Paper#3013]
### Link/paste of stack trace

Crash Report: https://pastebin.com/DJhLMQYn

### Plugin list:
dynmap
WildTP
WorldEdit
Playtime
ColoredSigns
LastLoginAPI
PluginManager
WorldBoarder
BuycraftX
Votifier
WOrldGuard
Multiverse-Core
AdvancedPortals
FastFarmStopper
KixsChatAnnouncer
sleep-most
PermsiionsEx
Vault
MyCommand
GriefPrevention
CoreProtect
WorldEditSelectionVisualizer
OreAnnouncer
ChestShop
AreaShop
Essentials
Parties
PerWorldInventory
VoteStreak
AdvancedAchievements
EssentialsChat
EssentialsGeoIP
SuperbVote
AFK-Kick
ServerSigns
EssentialsSpawn
Rankup

### Actions to reproduce (if known):
Crash Occured in Mesa Biome during mining gold Ore in an open Mineshaft. Rejoining after restart lead again to an instant crash. After Jailing myself and joining, the server kept going.

### Paper build number:
version git-Paper-129 1.15.2 API: 1.15.2-R0.1-SNAPSHOT

**Comment by Trigary** [comment]
When did the server crash for 2nd time? When you loaded that chunk by rejoining? Or did you try breaking the block again? Are you sure the cause of the crash is the breaking of a block?

Which of your plugins have anything to do with mining blocks/ores? Is OreAnnouncer one of them? Is that world a normal vanilla world or a Multiverse world?

Are the logs for the 1st or the 2nd crash?

**Comment by d00mfish** [comment]
Exactly, the server crashed by loading the chunk. I did not try to break the block again. I cant certanly say if it was the block break but I was in that region before about 10 minutes and there was no crash.

Yes OreAnnouncer checks for blockbreaks per minute and notifys any diamond find. And yes it was a multiverse world, even generated with mv.

The logs are for the first crash.
Here is the second crash: https://pastebin.com/033CnXnH
And the last and third: https://pastebin.com/80hsAYCm

After that I unloaded the world form multiverse and never joined it again.
